.changeset/breezy-oranges-impress.md

```diff
@@ -0,0 +1,6 @@
+---
+"@kaizen/draft-menu": major
+"@kaizen/select": major
```

Comment:
Small amendment to this, we just need to trigger a release for `@kaizen/draft-menu` so we can remove the @kaizen/select.

Also change a `minor` release, this doesn't constitute a breaking change to component functionality

Reply:
Thanks @mcwinter07, this has been updated.